test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -871,79 +845,59 @@ public static string CreateBasicTwoProjectSolution(TestDirectory workingPath, st
 EndProject
 Project(""{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}"") = ""proj2"", ""proj2\proj2.csproj"", ""{42641DAE-D6C4-49D4-92EA-749D2573554A}""
 EndProject");
+            CreateProject("proj1", proj1ConfigFileName, proj1Directory, new PackageIdentity("packageA", new NuGetVersion("1.1.0")));
+            CreateProject("proj2", proj2ConfigFileName, proj2Directory, new PackageIdentity("packageB", new NuGetVersion("2.2.0")));
 
-            var include1 = proj1ConfigFileName;
-
-            if (string.IsNullOrEmpty(include1))
+            // If either project uses PackageReference, then define "globalPackagesFolder" so the package doesn't get
+            // installed in the usual global packages folder.
+            if (proj1ConfigFileName.Equals("PackageReference") && redirectGlobalPackagesFolder)
             {
-                include1 = Guid.NewGuid().ToString();
+                CreateFile(workingPath, "nuget.config",
+@"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+  <config>
+    <add key=""globalPackagesFolder"" value=""GlobalPackages"" />
```

Comment:
Should this GPF name be unique per test-class or test-run or something? If multiple tests use the same GPF folder then would that defeat the purpose of this?:

>  so the package doesn't get
>             // installed in the usual global packages folder.